src/tests/comments.test.ts

```diff
@@ -102,11 +105,11 @@ describe("GET /comments", () => {
 
   describe("when there are comments", () => {
     beforeEach(async () => {
-      comments = comments.map((comment) => ({ ...comment, postID: post._id }));
+      comments = comments.map((comment: any) => ({ ...comment, postID: post._id }));
       comments.push({
-        content: "Third comment",
-        sender: "Benli",
         postID: "673b7bd1df3f05e1bdcf5321",
```

Comment:
If you give `comments` the type you should change this:
```suggestion
        postID: new mongoose.Schema.Types.ObjectId("673b7bd1df3f05e1bdcf5321"),
```